api/preprint_providers/serializers.py

```diff
@@ -19,6 +19,7 @@ class PreprintProviderSerializer(JSONAPISerializer):
     email_contact = ser.CharField(required=False, allow_null=True)
     email_support = ser.CharField(required=False, allow_null=True)
     example = ser.CharField(required=False, allow_null=True)
+    domain = ser.CharField(required=False, allow_null=True)
```

Comment:
Good catch here--also forgot to add the `domain_redirect_enabled` field, but fixed with the latest commit and updated views.py to match.